### DISCUSSION TITLE
Is there an easy to create a keymap alias?

### DISCUSSION THREAD

**Discussion by vdegenne** [LazyVim/LazyVim#4539] [Q&A] [answered]
I try to do that:
```lua
map("n", "s", "<leader>ss", opts)
```
But it doesn't work, the `<leader>` seems to be interpreted as a normal space.

I know I can create the key in the `keys` block of `fzf-lua` spec, but I'm trying to avoid code redundancy, and to see if there is a quick way to create aliases.

THanks

**Comment by dpetka2001** [comment, marked as answer]
`map("n", "s", "<leader>ss", { remap = true })` should do it I believe.

**Reply by vdegenne** [reply]
cool thx!
